coordinator/utils.go

```diff
@@ -0,0 +1,19 @@
+package coordinator
+
+import (
+	"time"
+)
+
+// hasPassed returns true if the provided duration has passed since the
```

Comment:
// hasPassed reports whether the provided duration
// added with the given time is before the current time or not.